waspc/packages/deploy/src/providers/fly/helpers/ICommonOps.ts

```diff
@@ -0,0 +1,42 @@
+import { cdToClientBuildDir, cdToServerBuildDir } from "./helpers.js"
+import * as tomlHelpers from "./tomlFileHelpers.js"
+
+export interface ICommonOps {
```

Comment:
I had higher hopes for this abstraction but only ended up using it for `cmd.ts`. :D All the others were different enough they couldn't easily be collapsed into a single function.